app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -77,4 +78,43 @@ abstract class AppDatabase : RoomDatabase() {
         """)
     }
   }
+
+  class Migration_4_5 : Migration(4, 5) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+      // Update local searchable name in the Patient table to strip out the newly added characters
+      database.inTransaction {
+        compileStatement("""update "Patient" set "searchableName"=? where "uuid"=?""")
```

Comment:
You don't need three quotes for single line statements right?

Reply:
I need it because I am using `"` for SQLite identifiers and I don't want to escape them in single line strings. I guess I could use backticks as well and make them single line strings.